overlord/configstate/config/helpers.go

```diff
@@ -279,8 +279,21 @@ type Conf interface {
 	State() *state.State
 }
 
+// ConfReader is an interface for reading of config values.
+type ConfReader interface {
+	Get(snapName, key string, result interface{}) error
+}
+
+// ApplyOptions encapsulates extra options passed to individual core config
+// handlers when configuration is applied to a specific root directory with
+// Apply().
+type ApplyOptions struct {
```

Comment:
can we move this to configcore? or am I missing something?

Reply:
Done.